public/give.html

```diff
@@ -157,6 +157,32 @@
         #map-stats, #list-stats {
           font-weight: 300;
         }
+
```

Comment:
Hey @patricknelson! Just diving in here so definitely don't have all the context, but wondering why we're including the CSS in a `style` tag instead of including a link to the css file? It might remove the need to copy the same set of styles into both html files.

Reply:
I don't like it either, it's the convention they went with. Not 100% sure, but @graemer  may have more perspective on that; reach out to them on our slack instance for more context.

And yeah, I agree. Right now it's very redundant and prone to errors (hence the need in this commit to restructure the HTML for consistency)!